Gemfile

```diff
@@ -7,6 +7,15 @@ gem 'net-imap' # for rspec
 gem 'net-pop'  # for rspec
 gem 'net-smtp' # to send email
 
+gem 'activestorage', '>= 6.1.7.7'
+gem 'activesupport', '>= 6.1.7.5'
+gem 'rails', '>= 6.1.7.5'
+
+gem 'omniauth', '>= 1.9', '< 3.0'
+
+gem 'nokogiri', '>= 1.16.2'
+gem 'rack', '>= 2.2.8.1'
+
```

Comment:
same ~> is preferable to >= because you can say 'anything in the major version rage'

Reply:
Cool, will go with your advice if it works!